### PR TITLE
added rich text into the readme, still needs the screenshot

### PR DESCRIPTION
**Pull request by WilliamHirschh**
Added rich text to README, Still needs screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# bird-tracker
-Repo for Project 1 - Create a Bird Tracker
+# Description
+    The bird sighting website was created to locate birds in any area and get information about them. Within this website you can locate birds and information along side it, aswell as creating your very own bird list.
```
Comment (mjkonkel) [inline README.md:2]: change 'aswell as' to 'as well as'

File: README.md [hunk 2]
```diff
@@ -1,2 +1,16 @@
...
+# Usage
+    To get started type in a city in the search box and hit search. Then you can see all the birds in that given location, if you want to see information about the bird click on the name and under the map some information will apear, to add it to the list just click the "Add to my list" button.
+# Screenshot
+
+# Links
+    https://mjkonkel.github.io/bird-tracker/
+# Authors
+    -Mitchell Konkel
+    -Nam Bui
+    -Van Meveren
```
Comment (mjkonkel) [inline README.md:13]: Tyler Van Meveren

File: README.md [hunk 3]
```diff
@@ -1,2 +1,16 @@
...
+    -William Hirsch
+# Licese
```
Comment (mjkonkel) [inline README.md:15]: License

File: README.md [hunk 4]
```diff
@@ -1,2 +1,16 @@
...
-Repo for Project 1 - Create a Bird Tracker
+# Description
+    The bird sighting website was created to locate birds in any area and get information about them. Within this website you can locate birds and information along side it, aswell as creating your very own bird list.
+
+# Usage
+    To get started type in a city in the search box and hit search. Then you can see all the birds in that given location, if you want to see information about the bird click on the name and under the map some information will apear, to add it to the list just click the "Add to my list" button.
+# Screenshot
+
+# Links
+    https://mjkonkel.github.io/bird-tracker/
```
Comment (mjkonkel) [inline README.md:9]: [Link to deployed application](https://mjkonkel.github.io/bird-tracker/)

Comment (mjkonkel) [inline README.md:9]: `[Link to deployed application](https://mjkonkel.github.io/bird-tracker/)`

**Review by nambui17 — APPROVED** [review]

**Pull request merged**